backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -899,11 +862,48 @@ public Set<Observation> convertToAOEResidenceObservation(
     return observations;
   }
 
+  public Set<Observation> convertToAOEGenderIdentityObservation(
+      String eventId, String genderIdentity) {
+    Set<Observation> observations = new LinkedHashSet<>();
+    if (StringUtils.isNotBlank(genderIdentity)
+        && GENDER_IDENTITIES.contains(genderIdentity.toLowerCase())) {
+      CodeableConcept genderIdentityLoincConcept =
+          createLoincConcept(LOINC_GENDER_IDENTITY, "Gender identity", "Gender identity");
+
+      String valueCode;
+      String valueDisplay;
+      String codeSystem = null;
+
+      switch (genderIdentity.toLowerCase()) {
+        case OTHER -> {
+          codeSystem = NULL_CODE_SYSTEM;
+        }
+        case REFUSED -> {
+          codeSystem = DATA_ABSENT_REASON_CODE_SYSTEM;
+        }
+        case FEMALE, MALE, NON_BINARY, TRANS_MAN, TRANS_WOMAN -> {
```

Comment:
Ccould we just use the "default" keyword here rather than enumerating the other "valid" values as a separate case? That way if we ever add a new value to the list of gender identities, we wouldn't also need to update it here.

Reply:
I think I would prefer to be explicit here.  Adding a valid value that is more in line with `OTHER` or `REFUSED` is just as reasonable of a future use case as adding a new gender value that would make sense with using default.

If anything I would be tempted to add a `case default -> { //throw some error }` just to make sure we don't miss making a change here when we need to.

Reply:
That default error case makes sense too: just want to account for the "additional value added" case and not have it fail silently.

Feel free to follow up in another PR however you decide

Reply:
opted to have default set the code to null and handle that instead - mostly because this is an optional field, and I wouldn't want to throw something that will break the fhir conversion.